### PR TITLE
Add renuo to „Who uses BEM #1“

### PR DESCRIPTION
**Pull request by aminagiger**
Thank you very much

### CODE REVIEW

File: src/assets/companies/renuo.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
...60 342.30 311.84 C 325.02 285.93 316.01 254.59 317.02 223.46 C 317.77 194.83 326.89 166.50 342.99 142.82 C 361.63 115.06 389.62 93.74 421.36 83.23 C 449.15 73.91 479.65 72.93 507.98 80.40 C 539.42 88.58 568.00 107.18 588.29 132.54 C 601.94 149.58 612.13 169.52 617.24 190.78 C 554.62 190.83 491.99 190.71 429.37 190.85 C 429.34 215.46 429.37 240.07 429.35 264.69 C 518.16 264.70 606.96 264.67 695.76 264.70 C 695.47 240.11 697.04 215.31 692.75 190.95 C 686.34 151.58 669.11 114.06 643.55 83.44 C 621.93 57.43 594.48 36.28 563.72 22.20 C 517.58 0.85 464.14 -4.08 414.84 8.28 C 378.06 17.35 343.70 35.98 315.92 61.73 C 269.71 104.00 242.21 166.24 243.25 228.97 C 243.05 268.49 254.07 307.90 274.03 341.95 C 295.31 378.27 326.90 408.47 364.16 428.07 C 394.33 444.04 428.18 452.83 462.28 454.00 L 0.00 454.00 L 0.00 417.31 C 24.53 417.33 49.05 417.27 73.58 417.35 C 73.80 353.23 73.57 289.11 73.70 224.99 C 74.32 195.08 83.86 165.42 101.07 140.93 C 119.61 114.27 146.81 93.70 177.63 83.40 C 193.29 78.08 209.83 75.44 226.36 75.41 C 226.34 50.80 226.37 26.19 226.35 1.59 C 185.29 1.57 144.27 12.87 109.20 34.25 C 70.01 57.88 38.30 93.68 19.61 135.45 C 7.88 161.47 1.33 189.74 0.00 218.24 L 0.00 0.00 Z" />
```
Comment (floatdrop) [inline src/assets/companies/renuo.svg]: Please, remove white background.

File: src/assets/companies/renuo.svg [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<path fill="#28448a" d=" M 1972.12 0.00 L 1984.74 0.00 C 2022.78 1.80 2060.40 13.10 2092.84 33.12 C 2124.51 52.49 2151.30 79.77 2170.09 111.80 C 2188.76 143.41 2199.34 179.66 2201.00 216.31 L 2201.00 236.70 C 2199.26 270.31 2190.22 303.60 2174.13 333.20 C 2157.15 364.65 2132.66 392.03 2103.19 412.29 C 2073.74 432.73 2039.32 445.80 2003.80 450.48 C 1992.04 438.89 1980.44 427.13 1968.71 415.50 C 1977.11 406.65 1985.93 398.20 1994.50 389.52 C 1999.61 384.66 2004.25 379.27 2009.63 374.71 C 2016.59 372.30 2023.88 370.75 2030.68 367.79 C 2075.16 350.48 2110.31 311.13 2122.26 264.88 C 2132.21 227.72 2127.25 186.87 2108.64 153.19 C 2092.90 124.33 2067.60 100.83 2037.65 87.28 C 2017.66 78.06 1995.64 73.76 1973.69 73.53 C 1985.25 61.17 1996.97 48.96 2008.67 36.73 C 1996.62 24.35 1984.26 12.29 1972.12 0.00 Z" />
+<path fill="#ffffff" d=" M 1984.74 0.00 L 2201.00 0.00 L 2201.00 216.31 C 2199.34 179.66 2188.76 143.41 2170.09 111.80 C 2151.30 79.77 2124.51 52.49 2092.84 33.12 C 2060.40 13.10 2022.78 1.80 1984.74 0.00 Z" />
```
Comment (floatdrop) [inline src/assets/companies/renuo.svg]: Please, remove white background.

**Comment by aminagiger** [comment]
Thanks, I have removed the background.

**Comment by floatdrop** [comment]
@aminagiger cool 🌟

**Comment by floatdrop** [comment]
:ship:

**Comment by aminagiger** [comment]
Thank you very much! Looks great.

**Pull request merged**